dgl_sparse/include/sparse/sparse_format.h

```diff
@@ -18,42 +18,71 @@
 namespace dgl {
 namespace sparse {
 
-/** @brief SparseFormat enumeration */
+/** @brief SparseFormat enumeration. */
 enum SparseFormat { kCOO, kCSR, kCSC };
 
-/** @brief CSR sparse structure */
+/** @brief COO sparse structure. */
+struct COO {
+  /** @brief the dense shape of the matrix. */
+  int64_t num_rows = 0, num_cols = 0;
+  /** @brief COO format row array of the matrix. */
+  torch::Tensor row;
+  /** @brief COO format column array of the matrix. */
+  torch::Tensor col;
+  /** @brief data index tensor. When it is null, assume it is from 0 to NNZ - 1.
+   */
+  torch::optional<torch::Tensor> data;
+  /** @brief whether the row indices are sorted. */
+  bool row_sorted = false;
+  /** @brief whether the column indices per row are sorted. */
+  bool col_sorted = false;
+};
+
+
+/** @brief CSR sparse structure. */
 struct CSR {
-  // CSR format index pointer array of the matrix
+  /** @brief the dense shape of the matrix. */
+  int64_t num_rows = 0, num_cols = 0;
+  /** @brief CSR format index pointer array of the matrix. */
   torch::Tensor indptr;
-  // CSR format index array of the matrix
+  /** @brief CSR format index array of the matrix. */
   torch::Tensor indices;
-  // The element order of the sparse format. In the SparseMatrix, we have data
-  // (value_) for each non-zero value. The order of non-zero values in (value_)
-  // may differ from the order of non-zero entries in CSR. So we store
-  // `value_indices` in CSR to indicate its relative non-zero value order to the
-  // SparseMatrix. With `value_indices`, we can retrieve the correct value for
-  // CSR, i.e., `value_[value_indices]`. If `value_indices` is not defined, this
-  // CSR follows the same non-zero value order as the SparseMatrix.
-  torch::optional<torch::Tensor> value_indices;
+  /** @brief data index tensor. When it is null, assume it is from 0 to NNZ - 1.
+   */
+  torch::optional<torch::Tensor> data;
```

Comment:
Why change it to `data`? `value_indices` is a more intuitive name.

Reply:
Just follow the name in old DGL. Will change them to `value_indices`.